packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -0,0 +1,217 @@
+const _ = require('lodash');
+const path = require('path');
+const colors = require('colors/safe');
+
+const { flags } = require('@oclif/command');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const { buildFlags } = require('../buildFlags');
+const utils = require('../../utils');
```

Comment:
instead of mass-importing `utils`, can you import the exact util files you need?